internal/launcher/analyze_test.go

```diff
@@ -6,138 +6,186 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestScriptArg(t *testing.T) {
+func TestIsScriptRequired(t *testing.T) {
```

Comment:
I think this should be pointing to "scriptNameFromArgs", as it's the method tested by this test suite.

Reply:
Fixed